charts/plugin-br-pix-direct-jd/templates/plugin-br-pix-direct-jd/configmap.yaml

```
@@ -8,8 +8,9 @@ data:
   # -- Default Environment variables for the plugin-br-pix-direct-jd
   NODE_ENV: {{ .Values.pix.configmap.NODE_ENV | default "production" | quote }}
   DEBUG: {{ .Values.pix.configmap.DEBUG | default "true" | quote }}
-  LOG_LEVEL: {{ .Values.pix.configmap.LOG_LEVEL | default "info" | quote }}
+  LOG_LEVEL: {{ .Values.pix.configmap.LOG_LEVEL | default "debug" | quote }}
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)